README.md

```diff
@@ -121,16 +113,10 @@ for more information about how to generate these.
 
 ```puppet
 node 'broker1.example.com' {
-  class { '::mcollective':
-    middleware         => true,
-    middleware_hosts   => [ 'broker1.example.com' ],
-    middleware_ssl     => true,
```

Comment:
Well, `middleware_hosts` and `middleware_ssl` *are* actually still around for client side configuration. Not sure we should strip the entire example out.